pkgs/applications/audio/zrythm/default.nix

```diff
@@ -61,48 +58,46 @@
 , serd
```

Comment:
`lilv`, `reproc`, `libcyaml`, and `libyaml` have been dropped as dependencies since the last version, so they can be removed here too.

Reply:
Edit - `libcyaml` is still required to upgrade old projects that are still in the yaml format.

Reply:
according to the cmake `lilv` is also still required to compile